src/django/users/models.py

```diff
@@ -23,41 +23,35 @@ class OrganizationTypes(object):
     """Enum-like object to track organization types"""
 
     ADMIN = 'ADMIN'
-    NEIGHBORHOOD = 'NEIGHBORHOOD'
     SUBSCRIBER = 'SUBSCRIBER'
 
     CHOICES = (
         (ADMIN, 'PFB Administrator Organization'),
-        (NEIGHBORHOOD, 'Neighborhood'),
         (SUBSCRIBER, 'Subscriber')
     )
 
 
 class Organization(PFBModel):
-    """Model for tracking neighborhood, subscribers, and admin groups
+    """Model for tracking subscribers and admin groups
 
 
     Every user will belong to an organization. Organization membership will determine
     whether or not a user can access or edit certain resources.
 
-    There are 3 types of organizations:
+    There are 2 types of organizations:
 
     **Admin.** Membership in the admin organization will grant access to all other organizations,
-    the ability to deactivate/activate neighborhood organizations, and the ability to create/delete
-    users within the system. Membership in this organization will be tightly controlled and will be
-    limited to PFB personnel who are responsible for activating/deactivating neighborhood sites and
-    creating initial admin users within a neighborhood organization.
-
-    **Neighborhood.** Membership in an organization that represents a neighborhood will only have
-    access to resources for that particular neighborhood.
+    and the ability to create/delete users within the system. Membership in this organization
+    will be tightly controlled and will be limited to PFB personnel who are responsible
+    for activating/deactivating neighborhood sites and creating initial admin users
+    within a neighborhood organization.
```

Comment:
As far as I can tell, we'll eventually need a structure where PFB is the only superuser org, and they are able to create other non-superuser organization accounts that they can hand out to the various cities. Either way, I don't think the third level is necessary. I think what we have here pretty much meets the requirements above, we may need to tweak the permissions classes farther down the line. For the April 1 deadline, that's not necessary, since we can run all jobs from the root org account.